cmd/infracost/cmd_test.go

```diff
@@ -133,8 +133,8 @@ func goldenFileCommandTest(t *testing.T, testName string, args []string, testOpt
 		_, err := os.Stat(hclFilePath)
 		if err == nil {
 			testutil.AssertGoldenFile(t, hclFilePath, actual)
+			return
```

Comment:
moved this here to return only if we find a `hcl` specific file. Having a return in the prior place was a bug as it was ignoring golden files in some HCL tests